FBSnapshotTestCase/FBSnapshotTestCasePlatform.m

```diff
@@ -34,7 +34,7 @@ BOOL FBSnapshotTestCaseIs64Bit(void)
 NSString *FBDeviceAgnosticNormalizedFileName(NSString *fileName)
 {
   UIDevice *device = [UIDevice currentDevice];
-  CGSize screenSize = [[UIApplication sharedApplication] keyWindow].bounds.size;
```

Comment:
My question about determines whether the solution should be _either_:

- Add an assertion and explain what users should do (set a host app or otherwise create a key window when visual tests are running)
- Create a `keyWindow` manually if necessary